images/ubuntu/hub/Dockerfile

```
@@ -11,7 +12,7 @@ RUN apt-get -q update \
 RUN sh -c 'echo "deb https://hub.unity3d.com/linux/repos/deb stable main" > /etc/apt/sources.list.d/unityhub.list' \
  && wget -qO - https://hub.unity3d.com/linux/keys/public | apt-key add - \
  && apt-get -q update \
- && apt-get -q install -y unityhub \
+ && apt-get -q install -y unityhub=$hubVersion \
```

I believe we need to quote this in case of unexpected inputs.

```suggestion
 && apt-get -q install -y "unityhub=$hubVersion" \
```